content/en/docs/contributing/release-process.md

```diff
@@ -291,4 +294,4 @@ The process for cutting a patch release is as follows:
 2. Then, continue with the instructions in [process for releasing a minor version](#process-for-releasing-a-minor-version).
 ## Links
```

Comment:
I propose to remove this section since it has nothing in it:

![empoty](https://user-images.githubusercontent.com/2195781/112636520-790d5c00-8e3d-11eb-90fb-c1ce3e431804.png)

```suggestion
```